frontend/awx/adapters/awxErrorAdapter.tsx

```diff
@@ -0,0 +1,40 @@
+import { ErrorOutput, GenericError } from '../../../framework/PageForm/typesErrorAdapter';
+import { isRequestError } from '../../common/crud/RequestError';
+
+export const awxErrorAdapter = (error: unknown): ErrorOutput => {
+  const genericErrors: GenericError[] = [];
+  const fieldErrors: GenericError[] = [];
+
+  if (
```

Comment:
This conditional feels overloaded. Is there a way to use a `Switch` statement that checks for `__all__ ` within `error.json` instead?